internal/storage/storage_handle_test.go

```diff
@@ -119,7 +119,6 @@ func (t *StorageHandleTest) TestNewStorageHandleWithZeroMaxConnsPerHost() {
 
 func (t *StorageHandleTest) TestNewStorageHandleWhenUserAgentIsSet() {
 	sc := getDefaultStorageClientConfig()
-	sc.UserAgent = "gcsfuse/unknown (Go version go1.20-pre3 cl/474093167 +a813be86df)"
```

Comment:
Sorry why did you remove this, and if this is necessary then I think this test case will become useless.

Reply:
getDefaultStorageClientConfig() method sets the same user agent. I've removed the test as it was redundant.